### ISSUE TITLE
# of active Golems of each type

### ISSUE THREAD

**Issue by Moedelawn** [PathOfBuildingCommunity/PathOfBuilding#2804]
**Is your feature request related to a problem? Please describe.**
When planning out a build utilizing large qualities of golems and golem buffs POB currently calculates as though I am summoning the maximum number of each golem. ex: I can have 9 maximum golems active, if i socket stone golem and flame golem POB will calculate the stats gained from having both of these golems as though i have 9 stone golems and 9 flame golems... which would be inaccurate

**Describe the solution you'd like**
Add a "Number of active golems" section to the configuration tab under "Skill options" with a number of each golem as you have them equipped as a numerical box
OR
in the "include in Full DPS" check where it says "Count:" increasing number of them there would work too

**Describe alternatives you've considered**
alternatively I would have to do a massive amount of math...

**Additional context**
love what you guys do, keep up the amazing work! <3

Edit 1: UNLESS IM 100% wrong and its only calculating 1 golem (still would be nice to have the setting to configure golem numbers)

Edit 2: ok I ran a bunch of math, I was initially under the impression it calculated for maximum golems active of each golem, it does not it only calculates 1 golem active... my original idea of being able to set number of golems per golem would I THINK still be a welcome addition to POB either way :3

**Comment by Wires77** [comment]
There is currently nothing that cares about the number of a specific type of golem in PoE.  PoB assumes you have the maximum number of golems summoned, regardless of the type composition.  If you're looking for the damage of your golems, you can include them in the FullDPS calculations and make use of the Count box, as you said.  If there's a specific situation that this doesn't cover, let us know, otherwise I will close this issue.

**Comment by Moedelawn** [comment]
for stats gained via buffs, Stone golem regeneration buff and flame golem increased damage buff and chaos golem physical damage reduction buff, but I only want 2 chaos golems, 2 stone golems and 5 flame golems... there is no way to currently calculate the stats gained by having a mix match of golems active. from my testing and understanding it only is currently showing the buffed stats of having 1 of each of these golems active at a time.

TLDR: not a golem build, using golems to utilize their buffs to buff my character, not using golems as a source of DPS. an assortment of golems granting various, buffs with different numbers of golems of each type active

**Comment by LocalIdentity** [comment]
Golem buffs do not stack, doesn't matter if you have 9 stone golems or just 1, you'll still have the same regen amount
